tests/beaker_tests/cisco_bgp/bgplib.rb

```diff
@@ -133,6 +138,10 @@ def self.create_bgp_manifest_present_non_default(platform)
         timer_bestpath_limit                   => '255',
         timer_bestpath_limit_always            => 'true',
         "
+    else
+      conditional_props <<
+        "# Nonstop Routing (NSR)
+        nsr                                    => 'false',"
```

Comment:
Since NSR defaults to false, this test (of non-default values) should be setting it to `true`.